main.go

```diff
@@ -165,38 +162,39 @@ func main() {
 	// resource objects.
 	if enableLease {
 		operatorNs, err := common.GetOperatorNamespace()
-		log.V(2).Info("Got operator namespace", "Namespace", operatorNs)
 		if err != nil {
-			if err == common.ErrNoNamespace || err == common.ErrRunLocal {
+			if errors.Is(err, common.ErrNoNamespace) || errors.Is(err, common.ErrRunLocal) {
 				log.Info("Skipping lease; not running in a cluster")
-			} else {
-				log.Error(err, "Failed to get operator namespace")
-				os.Exit(1)
 			}
-		} else {
 
-			log.Info("Starting lease controller to report status")
-			leaseUpdater := lease.NewLeaseUpdater(
-				clientset,
-				"config-policy-controller",
-				operatorNs,
-			)
-
-			//set hubCfg on lease updated if found
-			hubCfg, err := common.LoadHubConfig(hubConfigSecretNs, hubConfigSecretName)
-			if err != nil {
-				log.Error(err, "Could not load hub config, lease updater not set with config")
-			} else {
-				leaseUpdater = leaseUpdater.WithHubLeaseConfig(hubCfg, clusterName)
-			}
+			log.Error(err, "Failed to get operator namespace")
```

Comment:
It seems that the else should still be here since those errors in the `if` statement are ones to ignore.

Reply:
I think you're right. Nice catch.